### PR TITLE
Updates to load multi-site config

### PR DESCRIPTION
**Pull request by cobusc**

### CODE REVIEW

File: authentication_service/management/commands/demo_content.py [hunk 1]
```diff
@@ -34,15 +35,16 @@ def handle(self, *args, **options):
...
                 "client_secret": "super_client_secret_1",
                 "response_type": "code",
                 "jwt_alg": "HS256",
                 "redirect_uris": [
                     os.environ.get("WAGTAIL_1_CALLBACK", 'http://example.com/'),
                     os.environ.get(
                         "WAGTAIL_1_CALLBACK",
-                        'http://example.com/') + "register-redirect/"
+                        'http://example.com/') + "register-redirect/",
+                    os.environ.get("WAGTAIL_1_LOGOUT_REDIRECT", 'http://example.com/')
```
Comment (AltusBarry) [inline authentication_service/management/commands/demo_content.py]: Why is the logout redirect added to the `redirect_uri`s?

**Comment by AltusBarry** [comment]
+1, one question

File: authentication_service/management/commands/demo_content.py (hunk 1 above)
Comment (cobusc) [reply authentication_service/management/commands/demo_content.py]: The callback URLs look like this: `http://wagtail-demo-1-site-1:8000/oidc/callback/`
The logout URLs look like this: `http://wagtail-demo-1-site-1:8000/`, which happens to be the URLs that we redirect back to.

Comment (cobusc) [reply authentication_service/management/commands/demo_content.py]: I realise now that that "register-redirect/" URL looks incorrect. It computes to `http://wagtail-demo-1-site-1:8000/oidc/callback/register-redirect/` and from the `urls.py` in the `core-integration-demo` it looks like it should be `http://wagtail-demo-1-site-1:8000/register-redirect/`. I'll fix it.

**Pull request merged**